app/_locales/en/messages.json

```diff
@@ -1129,7 +1129,7 @@
     "message": "Confirm Secret Recovery Phrase"
   },
   "confirmTitleApproveTransaction": {
-    "message": "Allowance request"
+    "message": "Withdrawal request"
```

Comment:
We would need to remove this from other translations also

Reply:
Is this to make it re-translate it?